### ISSUE TITLE
How to specify the number of victim nodes which is being attacked?

### ISSUE THREAD

**Issue by thuwzy** [TaoShuchang/G-NIA#1]
I could not locate the line in the code to specify this. Thanks!

**Comment by TaoShuchang** [comment]
Hi thuwzy,
If you have a trained GNIA model, you can pass the target nodes directly to the "target" in the forward function of the GNIA class.
If you want to train a GNIA model that specifically attacks several target nodes, then you should provide data partitions for several targets, and ensure that the training set, validation set, and test set have no intersection with each other to avoid data leakage. The code of this division is updated in obtain_multi_targets of [utils.py](https://github.com/TaoShuchang/G-NIA/blob/master/utils.py). After running this function, the new division will be saved. When you train the model, please remember to use the newly saved update division.  You can adjust the number of target nodes by yourself. But if you attack too many target nodes, the training set may be quite small, and the performance will decrease when only injecting one node.

I will also update the running code of multiple targeted attacks in a few days.
Thank you for your attention.

**Comment by thuwzy** [comment]
Thank you for your reply!
